tests/backends/confluent_kafka.py

```diff
@@ -8,7 +8,44 @@
     TopicMetadata,
 )
 
-from snuba.perf import FakeConfluentKafkaMessage
+
+class FakeConfluentKafkaMessage(object):
```

Comment:
This is still used by things like the `StrictConsumer` which interact with the Confluent client directly.